src/components/FingerStatistics.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Box from "@mui/material/Box";
+
+type Hand = "left" | "right";
+
+type Finger = "thumb" | "first" | "second" | "third" | "fourth";
+
+type KeyData = {
+  keyName: string;
+  hand: Hand;
+  finger: Finger;
+  pushCount: number;
+  missCount: number;
+  timeSecCount: number;
+};
+
+type Props = {
+  data: KeyData[];
+};
+
+const FingerStatistics = (props: Props) => {
+  const { data } = props;
+  // キー毎の情報を指ごとの情報に変換する
+  // 後述のObject.keys()後のforEach()で型エラーにならないようにHandとFingerにstring型を追加しておく
+  const fingerData: {
+    [key1 in Hand | string]: {
+      [key2 in Finger | string]: { pushCountSum: number; missCountSum: number; timeSecSum: number };
```

Comment:
```suggestion
    [key1 in Hand]: {
      [key2 in Finger]: { pushCountSum: number; missCountSum: number; timeSecSum: number };
```
たしかに、Object.keys()の返り値の型はstring[]になるのですが、後述のように型を指定してあげるとstring型を追加せずに動作させることができます。